openquake/hmtk/tests/seismicity/catalogue_test.py

```diff
@@ -82,6 +84,15 @@ def setUp(self):
             [1960, 5.0],
         ])
 
+    def write_catalogue(self):
+        # Test the export of a catalogue in csv format
+        inp_file = 'data/test_write_input.csv'
+        out_file = 'data/test_write_output.csv'
```

Comment:
There is a trick to use. As it is, the test will break if you are not in the right directory. What you can do is to define

CURRENT_DIR = os.path.dirname(__file__)
and then 

￼        inp_file = os.path.join(CURRENT_DIR, 'data/test_write_input.csv')
￼        out_file = os.path.join(CURRENT_DIR, 'data/test_write_output.csv')